inst/shiny/modules/tab_nca/nca_results.R

```diff
@@ -132,15 +132,19 @@ nca_results_server <- function(id, pknca_data, res_nca, rules, grouping_vars, au
                                str_remove(col_names, "\\[.*"),
                                col_names)
 
-      final_results <- final_results() %>%
-        select(all_of(col_names[!(col_base_names %in% params_rem_cols)]))
+      final_results() %>%
+        select(c(all_of(col_names[!(col_base_names %in% params_rem_cols)])), -conc_groups)
+    })
+
+    output$myresults <- reactable::renderReactable({
```

Comment:
nitpick: while we are touching this line, we might remove the namespacing - it is not needed.
```suggestion
    output$myresults <- renderReactable({
```